content/en/logs/processing/attributes_naming_convention.md

```diff
@@ -25,66 +25,86 @@ This can generate confusion. For instance, a client IP might have the following
 
 The standard attribute panel pops when you add a new standard attribute or edit an existing one:
 
 {{< img src="logs/processing/attribute_naming_convention/define_standard_attribute.png" alt="Define Standard attribute"  style="width:80%;">}}
 
-Any element of the standard attributes can then be filled or updated.
 
-**Note**: Any updates or additions to standard attributes are only applied to newly ingested logs.
+## Standard attributes in Log Explorer
+
+
+### Aliased and Standard Facets display
+
+Typically, during a transitional period, standard facets may coexist in your organization alongside their aliased version(s). 
+
+The facet list provides users with visual hints to find their way through to the Standard Facets. And eventually update their preferred [saved views][22] to actually display Standard Facets rather than Aliased.
+
+{{< img src="logs/processing/attribute_naming_convention/aliased-to.gif" alt="Facet list highlight Naming Convention"  style="width:80%;">}}
 
-### Standard attribute remapping behavior
 
-After being processed in the pipelines, each log goes through the full list of standard attributes.
-For each entry of the standard attribute table, if the current log has an attribute matching the remapping list, the following is done:
+### Aliasing and Promoting
 
-* The first attribute that matches the provided list is remapped, and the value is overridden by the new one if already existing.
-* Datadog enforces the type of the remapped attribute. If this is not possible, the attribute is skipped and the next matching one of the list is used.
-* The original attribute is kept in the log.
+The facet list provides *admin* users with call-to-actions and hints for promoting facets, and aliasing facets towards standard facets. Promoting a facet as standard from the facet list creates a standard attribute whose type is the type of the facet.  
 
-**Important Note**: By default, the type of an existing standard attribute is unchanged if the remapping list is empty. Add the standard attribute to its own remapping list to enforce its type.
+{{< img src="logs/processing/attribute_naming_convention/alias-to_admin.png" alt="Curate Standard Attribute from Facet list"  style="width:80%;">}}
 
-#### Validation
+From the log side panel, *admin* users take alias any attribute (faceted, or non-faceted) towards a Standard Facet. 
 
-To add or update a standard attribute, follow these rules:
+{{< img src="logs/processing/attribute_naming_convention/alias-to_admin_panel.png" alt="Alias attribute from side panel"  style="width:80%;">}}
 
-* A standard attribute cannot be added in the remapping list of another standard attribute.
-* A custom attribute can be remapped to only one standard attribute.
-* To respect the JSON structure of the logs, it is not possible to have one standard attribute as the child of another (for example `user` and `user.name` cannot both be standard attributes).
 
 ## Default standard attribute list
```

Comment:
Would be great to move this into its own separate doc as the page is a little lengthy - mentioned in the docs channel but putting here as a reminder for when you make edits!